Atomic/backends/_docker_errors.py

```diff
@@ -0,0 +1,7 @@
+class DockerObjectNotFound(ValueError):
```

Comment:
_docker.py imports client.py; therefore, client.py cannot import from _docker.py.  Furthermore, several other files need access to these objects which would likely cause more circular issues.  I wasn't able offhand to figure out how to work around this with certainty other than breaking them out to their own file.